apps/freelance/src/pages/PasswordResetRequest/constants.ts

```diff
@@ -1,2 +1,7 @@
 export const emailName = 'email';
 export const infoFontSize = '1.1rem';
+export const errorFontSize = '0.9rem';
+export const errorColor = '#c93636';
```

Comment:
all colors should be in common file for full project, not for one component

Reply:
@lyudad fixed